amplify/backend/backend-config.json

```diff
@@ -274,295 +274,5 @@
       "providerPlugin": "awscloudformation",
       "service": "Lambda"
     }
-  },
```

Comment:
I think these changes are not intentional, right?

Reply:
Yup, changes reversed